web-client/tsconfig.json

```diff
@@ -24,6 +24,7 @@
     "enableI18nLegacyMessageIdFormat": false,
     "strictInjectionParameters": true,
     "strictInputAccessModifiers": true,
-    "strictTemplates": true
+    "strictTemplates": true,
+    "allowSyntheticDefaultImports": true
```

Comment:
I see the use of setting this to `true` - https://www.typescriptlang.org/tsconfig#allowSyntheticDefaultImports but I'm not sure why we need it.

Reply:
This property was added by the Angular CLI. 
it should cover the cases whereby we have things import like [this](https://github.com/ntls-io/nautilus-wallet/blob/5916aea15fd3a6920931a56e3a33667532189750/web-client/src/tests/transloco.helpers.ts)